ddl/ddl.go

```diff
@@ -1430,9 +1437,53 @@ func CancelJobs(se sessionctx.Context, ids []int64) (errs []error, err error) {
 	return cancelConcurrencyJobs(se, ids)
 }
 
+<<<<<<< HEAD
```

Comment:
Need to fix the conflict.

Reply:
Yes, but I wait https://github.com/pingcap/tidb/pull/43674 PR, it may handle it or have more conflicts